kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/create-domain.sh

```diff
@@ -321,6 +323,15 @@ function createYamlFiles {
     persistentVolumeClaimName=${domainUID}-weblogic-domain-pvc
   fi
 
+  # If logHome is not specified, defaults DOMAIN_LOGS_DIR to ${domainPVMountPath}/logs/${domainUID}
+  if [ -z "${logHome}" ]; then
+    domainLogsDir="${domainPVMountPath}/logs/${domainUID}"
```

Comment:
The default directory should not be setup here IMO - it should be be setup by the operator itself - the create-domain script can then leave the log dir blank/unset.